src/OrleansRuntime/Catalog/ActivationData.cs

```diff
@@ -193,6 +86,12 @@ public ActivationData(ActivationAddress addr, string genericArguments, Placement
         private ExtensionInvoker extensionInvoker;
         public IGrainMethodInvoker GetInvoker(int interfaceId, string genericGrainType=null)
         {
+            return GetInvoker(ref lastInvoker, ref extensionInvoker, interfaceId, genericGrainType);
+        }
+
+        public static IGrainMethodInvoker GetInvoker(ref IGrainMethodInvoker lastInvoker, ref ExtensionInvoker extensionInvoker, 
```

Comment:
Good point, will change to internal. Although the whole class is internal, so not a big deal.